server/command.go

```diff
@@ -435,6 +408,14 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 		return &model.CommandResponse{}, nil
 	}
 
+	if action == "help" {
+		if f, ok := p.CommandHandlers[action]; ok {
+			message := f(c, args, parameters, nil)
+			p.postCommandResponse(args, message)
+			return &model.CommandResponse{}, nil
+		}
+	}
```

Comment:
We can just call the help function directly here, instead of accessing `CommandHandlers`